package.json

```diff
@@ -12,6 +12,7 @@
   "scripts": {
     "build": "tsc",
     "commit": "commit",
+    "link": "yarn unlink; yarn build; yarn link",
```

Comment:
is this only being used for dev on this package and not expected to be used by the end user?

Reply:
Yeah, you're right.